data/contributors/organizations.json

```
@@ -489,25 +489,25 @@
       },
       {
         "username": "vmora",
-        "author_names": "Vincent Mora",
+        "author_names": "Vincent Mora,vmora",
         "from": "2013-01-01",
         "to": "2020-11-01"
       },
       {
         "username": "obrix",
-        "author_names": "Bertrand Rix",
+        "author_names": "Bertrand Rix,obrix",
```

```suggestion
        "author_names": "Bertrand Rix",
```

Same as previous comment